app/scripts/controllers/transactions/index.js

```diff
@@ -1219,13 +1219,24 @@ export default class TransactionController extends EventEmitter {
    *  params instead of allowing this method to generate them
    * @param options
    * @param options.estimatedBaseFee
+   * @param actionId
    * @returns {txMeta}
    */
   async createSpeedUpTransaction(
     originalTxId,
     customGasSettings,
     { estimatedBaseFee } = {},
+    actionId,
   ) {
+    // In transaction is found for same action id, do not create a new cancel transaction.
```

Comment:
```suggestion
    // If a transaction is found with the same actionId, do not create a new cancel transaction.
```
should `cancel transction` be `speed up transaction`?

Reply:
Somehow the suggestion is not committing seems to be a github issue. I added the change in a new commit.